docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -857,6 +857,99 @@
       </sect3>
     </sect2>
 
+    <sect2 id="Egress-HYAML" role="brk">
+      <title>Egress</title>
+
+      <para>In order to allow clusters to be securely locked down but still
+      allow access to the services they need. We have a similar mechanism to
+      Ingress that specifies what external endpoints it is permitted to
+      connect to egress. Keep in mind that Thor components do have their own
+      network policies.</para>
+
+      <para>For instance, in a default deployed system with network policies
+      enforced, a query running (on hThor, Thor, or Roxie), will not be able
+      to connect with a 3rd party service, such as an LDAP service or log
+      stack. Currently, allows any pod that happens to have been given Kube
```

Comment:
Reword sentence: 
This allows any pod with Kube API access, which doesn't have its own pod <emphasis>NetworkPolicy</emphasis> to access any external endpoint.